test/cmd/test-service/main.go

```diff
@@ -25,10 +25,6 @@ const (
 	EnvServiceCall = "SERVICE_CALL"
 )
 
-var (
-	rootDir = "test/cmd/test-service/assets/" //nolint[:deadcode]
```

Comment:
This is required to use the dev mode for assets (reading from fs)

Reply:
Ah ok, the comment would help here :)

Reply:
Fixed in beed499